example/FFTConvolutionOverRegionOfInterest.ipynb

```diff
@@ -0,0 +1,665 @@
+      "text/plain": [
+       "<Figure size 432x288 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Generate an input image and visualize ROI\n",
+    "\n",
+    "\n",
+    "def make_image() -> image_type:\n",
+    "    arr = np.ones([150, 150]).astype(np.float32)\n",
+    "    arr[20:80, 20:80] = 5\n",
+    "    arr[70:120, 60:100] = 0\n",
+    "    arr[30:60,50:120] = 10\n",
+    "    image = itk.image_from_array(arr)\n",
+    "    return image\n",
+    "\n",
+    "\n",
+    "def make_roi() -> region_type:\n",
+    "    region = itk.ImageRegion[2]()\n",
```

Comment:
Use `region_type` instead of `itk.ImageRegion[2]`?

Reply:
Amended